packages/react-table/src/components/TableComposable/Th.tsx

```diff
@@ -115,18 +119,32 @@ const ThBase: React.FunctionComponent<ThProps> = ({
         }
       })
     : null;
+  const collapseParams = collapse
+    ? collapsible(children as IFormatterValueType, {
+        rowData: {
+          isOpen: collapse.isCollapsed
```

Comment:
is this backward?. isOpen should be equivalent to isExpanded

Reply:
actually, this ended up being unused, removing